### PR TITLE
feat: create caregiver schedule page

### PR DESCRIPTION
**Pull request by Lilly-nn**
## Describe your changes

- I created schedule page for the caregiver where he/she can view their appointments for the specified date.
- 
image
image

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/CC-166)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [ ] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [ ] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [ ] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/caregiver-schedule/constants.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const currentDay = new Date();
```
Comment (nadiia-tsytsylina) [inline src/components/caregiver-schedule/constants.ts:1]: constants names should be lowercase.

**Review by nadiia-tsytsylina — APPROVED** [review]

File: src/pages/index.tsx [hunk 2]
```diff
@@ -5,21 +5,27 @@ import BookAppointment from 'src/components/book-appointment/BookAppointment';
 import MainHeader from 'src/components/reusable/header/MainHeader';
 import { useLocales } from 'src/locales';
 import { mockedAppointments } from 'src/components/appointments/helpers';
+import CaregiverSchedule from 'src/components/caregiver-schedule/CaregiverSchedule';
+import { useState } from 'react';
```
Comment (ProggerOleg) [inline src/pages/index.tsx:9]: import order

**Review by ProggerOleg — APPROVED** [review]

File: src/constants/index.ts [hunk 3]
```diff
@@ -7,6 +7,11 @@ export const MAX_CHARACTERS_LENGTH = 100;
 export const DATE_FORMAT = 'MM/dd/yyyy';
 export const BACKEND_DATE_FORMAT = 'yyyy-MM-dd';
 
+export const CURRENT_DAY = new Date();
+export const DISPLAY_DATE_FORMAT = 'dd MMMM, yyyy';
```
Comment (lyudad) [inline src/constants/index.ts:11]: у тебя на строке 7 уже есть форматирование даты, иметь разные форматирования по проекту не верно, нужно одно

Comment (Lilly-nn) [reply src/constants/index.ts:11]: А это по фигме так, когда мы показываем кеагиверу его расписание, в прикрепленном фото выше - 22 December, 2023; 7 строка это то как отображается дата в date picker компоненте

Comment (Lilly-nn) [reply src/constants/index.ts:11]: может тогда не ложить это в глобальные константы?

Comment (lyudad) [reply src/constants/index.ts:11]: не, это ты идешь к пму и говоришь, тут не хорошо, давай решим какой формат будем юзать

Comment (lyudad) [reply src/constants/index.ts:11]: это максимально не профессионально - говорить "так в фигме" когда в фигме фигня, а это часто

Comment (Lilly-nn) [reply src/constants/index.ts:11]: хорошо, поняла

File: src/components/caregiver-schedule/CaregiverAppointment.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+} from './styles';
+import { CaregiverAppointmentI } from './types';
+
+type Props = {
+  appointmentDays: CaregiverAppointmentI[];
+  openDrawer: () => void;
+};
+
+export default function CaregiverAppointment({ appointmentDays, openDrawer }: Props): JSX.Element {
+  const { translate } = useLocales();
```
Comment (lyudad) [inline src/components/caregiver-schedule/CaregiverAppointment.tsx:27]: empty line. fix eslint rules, if needed update rules

**Pull request merged**